unicorn/src/repository/zookeeper.cpp

```diff
@@ -0,0 +1,124 @@
+#include "zookeeper.hpp"
+
+#include <cocaine/context.hpp>
+#include <cocaine/logging.hpp>
+
+static int do_log(void* handler, const char* buf, size_t size) {
+    cocaine::logging::logger_t* log = static_cast<cocaine::logging::logger_t*>(handler);
+
+    std::string message(buf, size);
```

Comment:
Your only real mutation(except cutting) is tolower :-))) 
Anyway we don't expect much logs from ZK.